packages/router-core/tests/ssr-server-manifest.test.ts

```
@@ -1,9 +1,12 @@
 import { createMemoryHistory } from '@tanstack/history'
+import { runInNewContext } from 'node:vm'
```

ESLint

[error] `node:vm` import should occur before import of `@tanstack/history`

(import/order)